helix-loader/src/grammar.rs

```diff
@@ -146,8 +152,14 @@ pub fn fetch_grammars() -> Result<()> {
     Ok(())
 }
 
-pub fn build_grammars(target: Option<String>) -> Result<()> {
-    let grammars = get_grammar_configs()?;
+pub fn build_grammars(target: Option<String>, filter: Option<Vec<String>>) -> Result<()> {
+    let mut grammars = get_grammar_configs()?;
+    if let Some(filter) = filter {
+        if filter.len() > 0 {
+            grammars.retain(|grammar| filter.contains(&grammar.grammar_id));
+        }
+    }
```

Comment:
Why not just pass vec without option?

Reply:
The original thought process was that this would be a more explicit way of saying you won't filter anything. But looking now I have to agree, it makes more sense to just pass an empty vec instead.